File: CPP/Program/Vehicle.cpp
#pragma once
// Memasukan library yang digunakan
#include <iostream>

#include <string>


// Deklarasi namespace
using namespace std;

// Deklarasi kelas
class Vehicle
{
private: // Atribut Privat
    string platNomor;
    string merk;
    string tahunProduksi;
    string warna;

public:
    // Constructor
    Vehicle()
    {

    }
    //Constructor dengan Parameter
    Vehicle(string platNomor, string merk, string tahunProduksi, string warna)
    {
        this->platNomor = platNomor;
        this->merk = merk;
        this->tahunProduksi = tahunProduksi;
        this->warna = warna;
    }

    // Setter dan Getter
    void setPlatNomor(string platNomor)
    {
        this->platNomor = platNomor;
    }

    void setMerk(string merk)
    {
        this->merk = merk;
    }

    void setTahunProduksi(string tahunProduksi)
    {
        this->tahunProduksi = tahunProduksi;
    }

    void setWarna(string warna)
    {
        this->warna = warna;
    }

    string getPlatNomor()
    {
        return this->platNomor;
    }

    string getMerk()
    {
        return this->merk;
    }

    string getTahunProduksi()
    {
        return this->tahunProduksi;
    }

    string getWarna()
    {
        return this->warna;
    }

    // Destructor
    ~Vehicle()
    {

    }
};

// Vehicle::Vehicle(/* args */)
// {
// }

// Vehicle::~Vehicle()
// {
// }

File: Python/Program/Vehicle.py
## Saya Muhammad Rifky Afandi dengan NIM 2202346 mengerjakan LP2 dalam mata kuliah 
## Desain Pemorograman Berbasis Objek untuk keberkahanNya maka saya tidak 
## melakukan kecurangan seperti yang telah dispesifikasikan. Aamiin.

## Membuat class
class Vehicle:
    ## Atribut privat
    platNomor=""
    merk=""
    tahunProduksi=""
    warna=""
    
    ## Constructor
    ## Atribut Publik
    def __init__(self, platNomor="", merk="", tahunProduksi="", warna=""):
        self.platNomor = platNomor
        self.merk = merk
        self.tahunProduksi = tahunProduksi
        self.warna = warna

     ## Getter dan Setter
    def setPlatNomor(self, platNomor):
        self.platNomor = platNomor

    def setMerk(self, merk):
        self.merk = merk

    def setTahunProduksi(self, tahunProduksi):
        self.tahunProduksi = tahunProduksi

    def setWarna(self, warna):
        self.warna = warna

    def getPlatNomor(self):
        return self.platNomor

    def getMerk(self):
        return self.merk

    def getTahunProduksi(self):
        return self.tahunProduksi

    def getWarna(self):
        return self.warna
    